src/util/Vocabularies.ts

```diff
@@ -112,6 +113,21 @@ export const RDF = createUriAndTermNamespace('http://www.w3.org/1999/02/22-rdf-s
   'type',
 );
 
+export const SOLID = createUriAndTermNamespace('http://www.w3.org/ns/solid/terms#',
+  'oidcIssuer',
+  'oidcIssuerRegistrationToken',
+  'oidcRegistration',
+);
+
+export const SOLID_ERROR = createUriAndTermNamespace('urn:solid:error:',
+  'stack',
+);
+
+export const SOLID_HTTP = createUriAndTermNamespace('urn:solid:http:',
+  'location',
+  'slug',
+);
+
```

Comment:
Perhaps we should put those in `urn:npm:solid:community-server`?

Reply:
Sure, these were just added in the very beginning because we had no idea what to name them then.